tools/workbench/src/main/java/org/eclipse/rdf4j/workbench/util/QueryStorage.java

```diff
@@ -118,8 +118,9 @@ private boolean isShutdown() {
 	 * @throws RepositoryException if there is an issue creating the object to access the repository
 	 * @throws IOException
 	 */
-	protected QueryStorage(final AppConfiguration appConfig) throws RepositoryException, IOException {
```

Comment:
Would someone be extending or using this code in a way that they would need it to be protected?

Reply:
Highly unlikely - I suspect it was done like this because at the time we (mis)using protected all over the place...